app/questionnaire/path_finder.py

```diff
@@ -13,12 +18,18 @@
 
 
 class PathFinder:
-    def __init__(self, schema, answer_store, metadata, completed_blocks):
+    def __init__(
+        self,
+        schema: QuestionnaireSchema,
+        answer_store: AnswerStore,
+        metadata: Mapping,
+        completed: CompletedItems = CompletedItems(),
```

Comment:
`completed` should default to `None` rather than an instance. Currently if we have two instances of PathFinder, they will both use the same instance of CompletedItems?